pkg/cmd/util/clientcmd/factory_object_mapping.go

```diff
@@ -221,11 +222,15 @@ func (f *ring1Factory) LogsForObject(object, options runtime.Object, timeout tim
 
 func (f *ring1Factory) Scaler(mapping *meta.RESTMapping) (kubectl.Scaler, error) {
 	if deployapi.IsKindOrLegacy("DeploymentConfig", mapping.GroupVersionKind.GroupKind()) {
-		oc, kc, err := f.clientAccessFactory.Clients()
+		_, kc, err := f.clientAccessFactory.Clients()
```

Comment:
I don't like this. Are we planning to change `Clients()` to return openshift clientsets in the end? I do hope. Although probably it'll happen only when we actually remove `pkg/client`

Reply:
> I don't like this. Are we planning to change Clients() to return openshift clientsets in the end? I do hope. Although probably it'll happen only when we actually remove pkg/client

I think we will remove `Clients` altogether and create separate helpers for each individual type of client.